Tools/AP_Periph/Parameters.cpp

```diff
@@ -140,6 +140,12 @@ const AP_Param::Info AP_Periph_FW::var_info[] = {
     GOBJECT(servo_channels, "OUT",     SRV_Channels),
 #endif
 
+#ifdef HAL_PERIPH_ENABLE_NOTIFY
+    // @Group: NTF_
+    // @Path: ../libraries/AP_Notify/AP_Notify.cpp
+    GOBJECT(notify, "NTF_",  AP_Notify),
```

Comment:
shouldn't this be notify.lib?

Reply:
oh wow, yes. I wonder how this is even working!??!?!

Reply:
it doesn't compile as notify.lib. Also, battery is like this too. Hmm, interesting that this is the way it likes it.

Reply:
I've removed lib so this no linger applies